Return type annotation of init method should be None

Running `mypy --disallow-untyped-defs . | grep "__init__"` from the `src` directory returns:
```
naif_pds4_bundler/classes/log.py:21: error: The return type of "__init__" must be None  [misc]
naif_pds4_bundler/classes/setup.py:34: error: The return type of "__init__" must be None  [misc]
naif_pds4_bundler/classes/bundle.py:35: error: The return type of "__init__" must be None  [misc]
```

--- a/src/pds/naif_pds4_bundler/classes/bundle.py
+++ b/src/pds/naif_pds4_bundler/classes/bundle.py
@@ -31,7 +31,7 @@ class Bundle:
     :param setup: NPB execution Setup
     """
 
-    def __init__(self, setup) -> object:
+    def __init__(self, setup) -> None:
         """Constructor."""
         line = (
             f"Step {setup.step} - Bundle/data set structure generation "

--- a/src/pds/naif_pds4_bundler/classes/setup.py
+++ b/src/pds/naif_pds4_bundler/classes/setup.py
@@ -29,7 +29,7 @@ class Setup:
     :type version: str
     """
 
-    def __init__(self, args, version: str) -> object:
+    def __init__(self, args, version: str) -> None:
         """Constructor."""
         try:
             #

